src/DynamicHDT.cpp

```
@@ -18,23 +18,23 @@ std::string hdt::util::UInt32toString(uint32_t formID)
 	return std::string(buffer);
 }
 
-std::string _deprefix(std::string str_with_prefix)
+std::string _deprefix(std::string_view str_with_prefix)
```

Clang (14.0.6)

[warning] declaration uses identifier '_deprefix', which is reserved in the global namespace

(bugprone-reserved-identifier)